### PR TITLE
Refactor etcd certificates roles.

### PR DESCRIPTION
**Pull request by abutcher**

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

File: roles/etcd/meta/main.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ galaxy_info:
   description: etcd management
   company: Red Hat, Inc.
   license: Apache License, Version 2.0
-  min_ansible_version: 1.2
+  min_ansible_version: 1.9.4
```
Comment (detiber) [inline roles/etcd/meta/main.yml]: This should probably be 2.0, or really 2.1 now.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
@sdodson PTAL

File: library/delegated_serial_command.py [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (sdodson) [inline library/delegated_serial_command.py]: Add yourself to copyright, mention that it's derived from the ansible command module but not part of ansible?

File: library/delegated_serial_command.py [hunk 3]
```diff
@@ -0,0 +1,273 @@
...
+    -  If you want to run a command through the shell (say you are using C(<),
+       C(>), C(|), etc), you actually want the M(shell) module instead. The
+       M(command) module is much more secure as it's not affected by the user's
+       environment.
+    - " C(creates), C(removes), and C(chdir) can be specified after
+       the command. For instance, if you only want to run a command if
+       a certain file does not exist, use this."
+author:
+    - Ansible Core Team
+    - Michael DeHaan
```
Comment (sdodson) [inline library/delegated_serial_command.py]: here too i guess.

**Comment by sdodson** [comment]
LGTM, just the license comments for the locking module.

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Comment by abutcher** [comment]
aos-ci-test

**Pull request merged**